history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc

```diff
@@ -386,6 +395,10 @@ the lords were to go into a grand committee on the bill,
 but were prevented by an unexpected petition from some of the London clergy,
 which was presented by the Archbishop of York, and was as follows:
 
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 To the right honourable the Lords Spiritual and Temporal in Parliament assembled.
 
 The humble petition^
```

Comment:
So marking this as anything ( [.blurb], [.old-style], etc.) causes the footnote to be in the text in the PDF so I wasn't sure what to do.

Reply:
Yeah we have a problem with footnotes on headings.  Why don't you add `[.centered.emphasized]` above line 404 for now.  That should get it looking pretty good, and I made a [note to fix it](https://github.com/friends-library/friends-library/issues/85#issuecomment-636021593) whenever I come up with a way to handle footnotes in headings.